airflow/operators/latest_only_operator.py

```diff
@@ -32,6 +32,13 @@ class LatestOnlyOperator(BaseOperator):
     ui_color = '#e9ffdb'  # nyanza
 
     def execute(self, context):
+        # If the DAG Run is externally triggered, then return without
+        # skipping downstream tasks
+        if context['dag_run'].external_trigger:
+            logging.info("""Externally triggered DAG_Run:\n
```

Comment:
Nit: not sure about newline in log message. Usually this is poor form, as it breaks parsers and whatnot.